packages/core/js-peer/src/jsPeer/__test__/peer.spec.ts

```diff
@@ -55,7 +55,7 @@ describe('FluencePeer usage test suite', () => {
 
             expect(res).toBe('test');
         });
-    });
```

Comment:
how did you choose the tests that need this?

Reply:
It is the ones that did timeout because of marine-js performance. I will update marine-js today and remove timeouts